vite.config.ts

```diff
@@ -30,7 +30,7 @@ export default defineConfig({
   build: {
     lib: {
       entry: path.resolve(__dirname, 'src/lib/index.tsx'),
-      name: 'milkdown-datedata',
+      name: 'altos-react-text-editor',
```

Comment:
I saw, that you have changed the `name` in the `package.json`.
Note that it must be the same as in `vite.config.ts`. In addition, files that point to a particular filetype (UMD, Common.js) must also be tagged with this name.

I adjusted the `package.json` config to your new lib name.